### PR TITLE
Add onl-support command

### PR DESCRIPTION
**Pull request by pichuang**
`onl-support` command can dump useful information for ONL technical support.

Signed-off-by: Phil Huang <phil_huang@edge-core.com>

### CODE REVIEW

File: packages/base/all/vendor-config-onl/src/bin/onl-support [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+ch.setLevel(logging.INFO)
+logger.addHandler(ch)
+
+
+def copy_dir(src_dir):
+    dest = DIR_ROOT + src_dir
+    try:
+        copytree(src_dir, dest, symlinks=False, ignore=ignore_patterns('alternatives', '*.deb', '*.swi'))
+    except OSError as e:
+        if e.errno == errno.ENOTDIR:
```
Comment (TakeshiTseng) [inline packages/base/all/vendor-config-onl/src/bin/onl-support]: Make sure file exist there:

`if e.errno == errno.ENOTDIR and os.path.exist(src_dir): `

File: packages/base/all/vendor-config-onl/src/bin/onl-support [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+ch = logging.StreamHandler()
+ch.setLevel(logging.INFO)
+logger.addHandler(ch)
+
+
+def copy_dir(src_dir):
+    dest = DIR_ROOT + src_dir
+    try:
+        copytree(src_dir, dest, symlinks=False, ignore=ignore_patterns('alternatives', '*.deb', '*.swi'))
+    except OSError as e:
```
Comment (TakeshiTseng) [inline packages/base/all/vendor-config-onl/src/bin/onl-support]: Some files might throw "No such file or directory" error

e.g. /etc/rc0.d/K02ofdpa

**Comment by jnealtowns** [comment]
Thanks for adding this. 
It would be a bit better if this were implemented as an onl.support python module. 
If you made an OnlSupport class and added all of the directory path variables as class variables then other tools could import the OnlSupport class and access any collateral as well. 

The ``onl-support`` tool would then just ``from onl.support import OnlSupport`` and run ``OnlSupport.main()``

**Comment by pichuang** [comment]
Hi @jnealtowns Could you please help me to review the code?

**Comment by sonoble** [comment]
Hi,
I appologize but Jeff is not available right now.  If I am able to find an engineer to look at the code I will.  Thank you again for the contribution, it will be great to see it as part of ONL.

**Comment by pichuang** [comment]
@sonoble Sure. It's OK. Thanks you for quick reply :)

File: packages/base/all/vendor-config-onl/src/python/onl/support/OnlSupport.py [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+import errno
+import os
+import subprocess
+import time
+
+from onl.platform.current import OnlPlatform
+
+copy_dir_list = ['/etc',
+                 '/var/log',
+                 '/mnt/onl']
```
Comment (jnealtowns) [inline packages/base/all/vendor-config-onl/src/python/onl/support/OnlSupport.py]: This looks like you are trying to copy all of /mnt/onl/* into /tmp/*. This is going to blow up.

**Comment by jnealtowns** [comment]
@pichuang Do you plan to address the issue of attempting to copy the mount points into the tech support?

**Comment by jnealtowns** [comment]
Closing due to inactivity. Please resubmit for review when ready.

**Pull request closed without merging**